inspirehep/modules/theme/static/js/settings.js

```diff
@@ -69,6 +71,7 @@ require.config({
     'profile': 'js/authors/profile',
     'publications': 'js/authors/publications',
     'readmore': 'node_modules/readmore-js/readmore',
+    'search_parser': 'node_modules/inspirehep-typeahead-search-js/src/search_parser',
```

Comment:
If I understood this right you're reordering alphabetically this configuration. That's :+1:, but I will split this in a preparatory commit.